queries.sql

```diff
@@ -7,4 +7,64 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+--setting the species column to unspecified
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+--Update the species column to digimon for all animals that have a name ending in mon
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species = 'NULL' OR species = '';
+SELECT * FROM animals;
+
+COMMIT;
+SELECT * FROM animals;
+
+--delete all records in the animals table
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+
+--Delete all animals born after Jan 1st, 2022.
+BEGIN;
+DELETE FROM animals
+WHERE date_of_birth  > '2022-01-01';
+--
+BEGIN
+SAVEPOINT first_save;
+--
+
+UPDATE animals
+SET weight_kg  = weight_kg  * -1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT neutered, MIN(weight_kg ), MAX(weight_kg)
+FROM animals
+GROUP BY neutered;
```

Comment:
- [ ] Great job working on the queries so far but I think this question asks What is the minimum and maximum weight of each type of animal? so I think it would be better if you fix it from select neutered to species to select each type ⭐🚀💯